packages/stg-evm-usdt-v2/hardhat.config.js

```diff
@@ -0,0 +1,109 @@
+require('@nomicfoundation/hardhat-toolbox');
+require('@openzeppelin/hardhat-upgrades');
+require('dotenv').config();
+
+const accounts = {
+    mnemonic: process.env.MNEMONIC,
+};
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+    namedAccounts: {
+        deployer: {
+            default: 0,
+        },
+        alice: {
+            default: 1,
+        },
+        bob: {
+            default: 2,
+        },
+        carol: {
```

Comment:
*carrot

Reply:
Oo i think this just may be the name `carol` but if you really like orange vegetables then I am willing to make this significant change for you.